src/nisarqa/processing/non_insar/range_spectra.py

```diff
@@ -111,7 +111,8 @@ def generate_range_spectra_single_freq(
     with product.get_raster(freq, first_pol) as img:
         # Compute the sample rate
         # c/2 for radar energy round-trip; units for `sample_rate` will be Hz
-        dr = product.get_slant_range_spacing(freq)
+        dr = product.get_slant_range_posting(freq)
+        dr = abs(dr)
```

Comment:
As mentioned above, let's revert this from "posting" to "spacing". Then, since we know that slant range spacing is always positive, no need to take the absolute value.

```suggestion
        dr = product.get_slant_range_spacing(freq)
```